File: mineGame/TitleViewController.m
//
//  TitleViewController.m
//  mineGame
//
//  Created by masapp on 2015/05/03.
//  Copyright (c) 2015年 masapp. All rights reserved.
//

#import "TitleViewController.h"
#import "GameViewController.h"

@interface TitleViewController ()

@end

@implementation TitleViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.

    UIImageView *title = [[UIImageView alloc] initWithFrame:[[UIScreen mainScreen] applicationFrame]];
    title.image = [UIImage imageNamed:@"title.png"];
    [self.view addSubview:title];
}

- (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
{
    GameViewController *gameViewController = [[GameViewController alloc] init];
    [self presentViewController:gameViewController animated:NO completion:nil];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOS/mineGame/Classes/TitleViewController.swift
//
//  TitleViewController.swift
//  mineGame
//
//  Created by masapp on 2018/04/16.
//  Copyright © 2018 masapp. All rights reserved.
//

import UIKit

class TitleViewController: UIViewController {
    
    // MARK: - UIViewController
    override func viewDidLoad() {
        let title = UIImageView(frame: UIScreen.main.bounds)
        title.image = UIImage(named: "title")
        self.view.addSubview(title)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let gameVC = GameViewController()
        self.present(gameVC, animated: false, completion: nil)
    }
}
